tests/compile_ouroboros.py

```diff
@@ -0,0 +1,670 @@
+# -*- coding: utf-8 -*-
+"""Check modules from Ouroboros that compile with voc."""
+
+# Standard library imports
+from datetime import datetime
+from subprocess import Popen, PIPE
+import multiprocessing
+import os
+
+HERE = os.path.dirname(os.path.realpath(__file__))
+MODULES_WHITELIST = [
```

Comment:
One other nitpick - there needs to be a mode for overrides. While in many cases we will be able to just compile the `.py` file from ouroboros and use it, for performance we may sometimes use a native Java implementation of a module. So - if there's a native module in the `python` subdirectory, the ouroboros version should be skipped.